python/katana/example_data.py

```diff
@@ -54,15 +61,24 @@ def get_input(rel_path) -> Path:
     >>> from katana.local import Graph
     ... graph = Graph(get_input("propertygraphs/ldbc_003"))
     """
-    path = get_inputs_directory() / rel_path
+    path = get_inputs_directory(rel_path=rel_path) / rel_path
     if path.exists():
         return path
     return get_inputs_directory(invalidate=True) / rel_path
 
 
-def get_input_as_url(rel_path) -> str:
+def get_input_as_url(rel_path, datasource="local") -> str:
     """
     Similar to get_input, but return the graph as file:// URL.
     """
+    
+    if datasource == "gcp":
+        urls = {"gcp": {"propertygraphs/ldbc_003": "gs://katana-demo-datasets/rdg-datasets/ldbc/v1/ldbc_003_",
+                "external/csv-datasets": "gs://katana-demo-datasets/csv-datasets",
+                "propertygraphs/rmat10": "gs://katana-demo-datasets/rdg-datasets/rmat/rmat10",
+                "propertygraphs/rmat10_symmetric": "gs://katana-demo-datasets/rdg-datasets/rmat/rmat10_symmetric"}
```

Comment:
Is `gs://katana-demo-datasets` public? If it's listed here it needs to be.

Reply:
The tests will be run against a katana cluster deployed in cloud and these csv/rdg datasets are used for generating graphs and queries.

Reply:
Moved them to enterprise.